api/envoy/config/filter/http/jwt_authn/v2alpha/config.proto

```diff
@@ -390,7 +390,8 @@ message FilterStateRule {
 
   // A map of string keys to requirements. The string key is the string value
   // in the FilterState with the name specified in the *name* field above.
-  map<string, JwtRequirement> requires = 3;
+  map<string, JwtRequirement>
```

Comment:
?

Reply:
Huh, that is strange. I think this happened while running proto format, I'll revert the JWT files.